frontend/src/metabase/parameters/components/ValuesSourceModal/ValuesSourceModal.unit.spec.tsx

```diff
@@ -48,14 +48,6 @@ describe("ValuesSourceModal", () => {
   const field2 = checkNotNull(metadata.field(2));
 
   describe("fields source", () => {
-    it("should show a message about not connected fields", async () => {
```

Comment:
This test is removed on purpose, because we no longer show this message in the UI with the "From connected fields" option. The new behaviour is on [line 79](https://github.com/metabase/metabase/pull/34958/files#diff-0608162840ea3b5a4d59891e96645e7288f22470875f6b43a075771a5e4e7d73R79)